### PR TITLE
Replace json elements

### PR DESCRIPTION
**Pull request by SimonRubenDrauz**
if e.g. a path changes, you can pass a dict to replace the changed elements

### CODE REVIEW

File: pandapower/file_io.py [hunk 1]
```diff
@@ -302,6 +308,10 @@ def from_json_string(json_string, convert=False, encryption_key=None, elements_t
         >>> net = pp.from_json_string(json_str)
 
     """
+    if not replace_elements is None:
```
Comment (rbolgaryn) [inline pandapower/file_io.py:311]: # better would be:
if replace_elements is not None:

**Review by rbolgaryn — CHANGES_REQUESTED** [review]
please add information to CHANGELOG

File: pandapower/file_io.py (hunk 1 above)
Comment (SimonRubenDrauz) [reply pandapower/file_io.py:311]: thanks. Adapted it.

Comment (SimonRubenDrauz) [reply pandapower/file_io.py:311]: added a changelog

File: pandapower/test/api/test_file_io.py [hunk 2]
```diff
@@ -427,5 +429,28 @@ def test_json_io_with_characteristics(net_in):
...
+
+    net_load = pp.from_json_string(json_string,
+                                   replace_elements={r'pandapower.control.controller.const_control':
+                                                         r'pandapower.test.api.input_files.test_control'})
+    assert net_orig.controller.at[0, 'object'] == net_load.controller.at[0, 'object']
+    assert nets_equal(net_orig, net_load)
+    pp.runpp(net_load, run_control=True)
+    pp.runpp(net_orig, run_control=True)
+    assert (net_load.controller.loc[0, 'object'].check_word == 'banana')
+    assert not net_orig.controller.at[0, 'object'] == net_load.controller.at[0, 'object']
```
Comment (rbolgaryn) [inline pandapower/test/api/test_file_io.py:451]: != would be better

**Review by rbolgaryn — CHANGES_REQUESTED** [review]

File: pandapower/test/api/test_file_io.py (hunk 2 above)
Comment (SimonRubenDrauz) [reply pandapower/test/api/test_file_io.py]: Agree. Adapted it!

**Review by rbolgaryn — APPROVED** [review]

**Pull request merged**